src/java.base/share/classes/java/lang/StrictMath.java

```diff
@@ -309,8 +309,10 @@ public static double log10(double a) {
      * @param   a   a value.
      * @return  the positive square root of {@code a}.
      */
-    @IntrinsicCandidate
-    public static native double sqrt(double a);
+    // @IntrinsicCandidate -- Temporarily disable for testing
```

Comment:
I'll revert commenting out the intrinsic annotation before pushing.

Reply:
> I'll revert commenting out the intrinsic annotation before pushing.

Turns out some VM updates are needed to preserve the intrinsic if this method declaration is changed; I'll be contacting the HotSpot team for assistance.